susemanager/src/mgr_bootstrap_data.py

```diff
@@ -34,6 +34,7 @@
 
 PKGLIST11 = [
     "dbus-1-python",
+    "dmidecode",
```

Comment:
I just checked on SLE11SP4 tree and I cannot find the dmidecode RPM.
Am I missing anything?

Reply:
@mbologna you are right, I wonder how `python-dmidecode` (line 48) works. Let me review this again.

Reply:
Fixed. As Pablo said, it's pmtools, but does not have 'provides', so I had to change it here and then make a workaround at the `bootstrap.sls` file

Reply:
https://build.suse.de/package/view_file/SUSE:SLE-11-SP3:Update/pmtools/pmtools.spec?expand=1
https://build.suse.de/package/view_file/SUSE:SLE-11-SP4:Update/pmtools/pmtools.spec?expand=1